tools/rna_tools/vienna_rna/rnalfoldz.xml

```diff
@@ -0,0 +1,97 @@
+<tool id="rnalfoldz" name="RNALfoldz" version="2.1.6.0">
+    <description>calculates locally stable secondary structures of RNA</description>
+    <expand macro="requirements" />
+    <expand macro="version_command" />
+    <expand macro="stdio" />
+    <macros>
+        <token name="@EXECUTABLE@">RNALfold</token>
+        <import>macros.xml</import>
+    </macros>
+    <command>
+<![CDATA[
+        RNALfold -L $span -T $temperature -d$dangling -z$zthresh < $fasta_input | sed -r 's/(^[.()]*) +\((.*)\) +(.*) +z= +(.*)/\1\t\2\t\3\t\4/' > $out_file
+        #if $varExists('$advancedOptions.noconversion')
+            #if $advancedOptions.noconversion
+                --noconv
+            #end if
+            #if $advancedOptions.gquad
+                --gquad
+            #end if
+            #if $advancedOptions.nolp
+                --noLP
+            #end if
+            #if $advancedOptions.nogu
+                --noGU
+            #end if
+            #if $advancedOptions.noclosinggu
+                --noClosingGU
+            #end if
+            #if $advancedOptions.notetra
+                --noTetra
+            #end if
+        #end if
+]]>
+    </command>
+    <inputs>
+	<param format="fasta" name="fasta_input" type="data" label="FASTA file"/>
+        <param name="zthresh" size="6" type="float" value="0" label="threshold for z-score computation (-z)" help="-z"/>
```

Comment:
min? max? default (seems to be -2 https://www.tbi.univie.ac.at/RNA/RNALfold.1.html)?